colin-api/src/colin_api/utils/__init__.py

```diff
@@ -61,6 +61,22 @@ def convert_to_pacific_time(thedate: str) -> str:
         raise err
 
 
+def convert_to_date_pacific_time(thedate: str) -> str:
+    """Convert the datetime string to date only pacific time string."""
+    try:
+        # tries converting two formats before bailing
+        try:
+            datetime_obj = datetime.datetime.strptime(thedate, '%Y-%m-%dT%H:%M:%S.%f%z')
+        except Exception:  # pylint: disable=broad-except;
+            datetime_obj = datetime.datetime.strptime(thedate, '%Y-%m-%dT%H:%M:%S%z')
+        datetime_utc = datetime_obj.replace(tzinfo=timezone('UTC'))
+        datetime_pst = datetime_utc.astimezone(timezone('US/Pacific'))
```

Comment:
Since the TZ info is starting to crop up in a number of areas, either extract the text to a Final, or pull to common utility function.

Reply:
ok. I created some constants for TZ.